src/encoded/static/components/forms/SubmissionView.js

```diff
@@ -1072,39 +1081,41 @@ export default class SubmissionView extends React.PureComponent{
                     }
                 }
 
-                // if testing validation, use check_only=True (see /types/base.py)
-                var destination = test ? '/' + currType + '/?check_only=True' : '/' + currType,
+                // if testing validation, use check_only=True (see /types/base.py)'
+                let destination;
+                let actionMethod;
+                // used to keep track of fields to delete with PATCH for edit/round two; comma-separated string
+                let deleteFields;
+                // change actionMethod and destination based on edit/round two
+                if (this.state.roundTwo){
+                    destination = this.state.keyComplete[inKey];
+                    actionMethod = 'PATCH';
+                    const alreadySubmittedContext = this.state.keyContext[destination];
+                    // roundTwo flag set to true for second round
+                    deleteFields = this.buildDeleteFields(finalizedContext, alreadySubmittedContext, currSchema);
+                } else if (this.state.edit && inKey === 0){
+                    destination = object.itemUtil.atId(propContext);
+                    actionMethod = 'PATCH';
```

Comment:
I think we need `deleteFields = this.buildDeleteFields(finalizedContext, propContext, currSchema);` here